flake/packages.nix

```diff
@@ -31,7 +31,7 @@
         lib.mkMerge [
           testbedPackages'
           {
-            docs = import "${self}/docs" { inherit pkgs inputs lib; };
+            docs = lib.modules.importApply "${self}/docs" { inherit pkgs inputs lib; };
```

Comment:
> do you know this is causing the docs to fail?

Looks like this change needs reverting.

This isn't a module, so it shouldn't use `importApply`.

This would be better off using `pkgs.callPackage`